pkg/controllers/dynakube/version/conditions.go

```diff
@@ -0,0 +1,31 @@
+package version
+
+import (
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	verifiedReason            = "Verified"
+	verificationSkippedReason = "VerificationSkipped"
+)
+
+func setVerifiedCondition(conditions *[]metav1.Condition, conditionType string) {
+	condition := metav1.Condition{
+		Type:    conditionType,
+		Status:  metav1.ConditionTrue,
+		Reason:  verifiedReason,
+		Message: "Version verified for component.",
+	}
+	_ = meta.SetStatusCondition(conditions, condition)
+}
+
+func setVerificationSkippedReasonCondition(conditions *[]metav1.Condition, conditionType string) {
```

Comment:
```suggestion
func setVersionUpdateSkippedReasonCondition(conditions *[]metav1.Condition, conditionType string) {
```

Reply:
the version update wasn't necessarily skipped, because if the custom version/image was changed we would still set the status to be `VerificationSkipped`, because we didn't verify anything

the name is not the best but I tried to convey the fact that if custom stuff is used then we will just "go with it", instead of trying to be smart about it